### PR TITLE
RPP Test Suite - Function Group Modification

### PR DESCRIPTION
**Pull request by HazarathKumarM**
- Add new augmentation group map
- Modified function group finder function

### CODE REVIEW

File: utilities/test_suite/HIP/runTests.py [hunk 1]
```diff
@@ -51,26 +51,22 @@ def get_log_file_list(preserveOutput):
 
 # Functionality group finder
 def func_group_finder(case_number):
-    if case_number < 5 or case_number == 13 or case_number == 36 or case_number == 45:
```
Comment (r-abishek) [inline utilities/test_suite/HIP/runTests.py:54]: Please remove all this in L53-L69. This is all you need:
```
def func_group_finder(case_number):
    for key, value in ImageAugmentationGroupMap.items():
        if case_number in value:
            return key
    return "miscellaneous"
```

File: utilities/test_suite/HOST/runTests.py [hunk 2]
```diff
@@ -51,19 +51,19 @@ def get_log_file_list(preserveOutput):
 
 # Functionality group finder
 def func_group_finder(case_number):
-    if case_number < 5 or case_number == 13 or case_number == 36 or case_number == 31 or case_number == 45:
```
Comment (r-abishek) [inline utilities/test_suite/HOST/runTests.py:54]: If this func_group_finder() is repeated between HIP and HOST, please put it in common.py

**Review by r-abishek — CHANGES_REQUESTED** [review]
@HazarathKumarM Pls address comments

File: utilities/test_suite/HOST/runTests.py (hunk 2 above)
Comment (HazarathKumarM) [reply utilities/test_suite/HOST/runTests.py:54]: Done

File: utilities/test_suite/HIP/runTests.py (hunk 1 above)
Comment (HazarathKumarM) [reply utilities/test_suite/HIP/runTests.py:54]: Done

**Review by r-abishek — APPROVED** [review]

**Pull request merged**